sms-worker@.service.template

```diff
@@ -0,0 +1,49 @@
+[Unit]
+Description=sms-worker
+After=docker.service
+Requires=docker.service
+After=consul@%i.service
+Wants=consul@%i.service
+After=rabbitmq@%i.service
+Requires=rabbitmq@%i.service
+
+[Service]
+EnvironmentFile=/etc/environment
+TimeoutStartSec=10m
+TimeoutStopSec=10m
+
+Restart=on-failure
+
+Environment=DOCKER_IMAGE=
+Environment=CONTAINER=sms-worker
+Environment=HOME=/root
+
+ExecStartPre=-/usr/bin/docker kill ${CONTAINER}
+ExecStartPre=-/usr/bin/docker rm ${CONTAINER}
+ExecStartPre=/usr/bin/docker pull ${DOCKER_IMAGE}
+
+ExecStartPre=/bin/bash -c 'echo ACCESS_KEY_ID="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/aws/access-key-id?raw)" > /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo SECRET_ACCESS_KEY="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/aws/secret-access-key?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo QUEUE_PREFIX="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/aws/queue-prefix?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo DB_PREFIX="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/aws/db-prefix?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo ENVIRONMENT="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/environment?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo CIVIC_API_KEY="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/civic-api/key?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo CIVIC_API_ELECTION_ID="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/civic-api/election-id?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo CIVIC_API_OFFICIAL_ONLY="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/civic-api/official-only?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo TWILIO_SID="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/twilio/sid?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo TWILIO_TOKEN="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/twilio/token?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo TWILIO_NUMBER="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/twilio/number?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo PROCS="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/procs?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo ROUTINES="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/routines?raw)" >> /tmp/${CONTAINER}--env'
+ExecStartPre=/bin/bash -c 'echo LOGGLY_TOKEN="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/sms/loggly-token?raw)" >> /tmp/${CONTAINER}--env'
```

Comment:
Do we still need this `loggly-token`?  Are we not fully converted to papertrail on the SMS app yet?

Reply:
Tested without it on staging, seemed ok, so I removed it.